hugo/layouts/partials/tags.html

```diff
@@ -1,15 +1,20 @@
-{{ $tags:= .tags| default slice }}
+{{ $context := .context | default slice }}
+{{ $tags := $context.Params.tags | default slice }}
+{{ $tagOptions := $context.Site.Params.Tags | default slice }}
+{{ $filteredTags := where $tagOptions "name" "in" $tags | default slice }}
```

Comment:
I would make it so the tags partial doesn't have to do filtering but gets the full tags allready. This way the partial is better reusable in all cases. How it gets the tags logic should not be inside the partial in my opinion